cloudformation_templates/aws_digitalmarketplace_admin_frontend.json

```diff
@@ -156,6 +186,10 @@
       "Description": "Digital Marketplace EB environment",
       "Value": {"Ref": "Environment"}
     },
+    "Subdomain": {
+      "Description": "The subdomain of the app",
+      "Value": {"Ref": "Subdomain"}
+    },
     "URL": {
```

Comment:
We should return the URL with the new domain here, since it's used to link the apps together. We'll just have to switch it to https when the certificates is in place.